JS/validation.js

```diff
@@ -0,0 +1,12 @@
+const conForm = document.querySelector('.ContForm-cont');
+const mail = document.getElementById('email');
+
+conForm.addEventListener('submit', (e) => {
+  if (mail.value !== mail.value.toLowerCase()) {
+    e.preventDefault();
+    const emailLC = mail.value.toLowerCase();
+    mail.value = emailLC;
+    // eslint-disable-next-line no-alert
+    alert('Please enter your email in all lowercase, we do it for you only check it if it is ok');
```

Comment:
Instead of an alert, an error message should be shown to the user near the submit button informing them of the error, and the form is not sent because it's part of the project's requirement 👍

Reply:
Solved